### DISCUSSION TITLE
Transmitter calibration?

### DISCUSSION THREAD

**Discussion by jclagreca** [egzumer/uv-k5-firmware-custom#168] [General]
I've seen in some other [forums](https://www.facebook.com/groups/961148971585971/permalink/1085902849110582) where they have a firmware JUST for transmitter calibration.  I'm not sure how much space this option would consume, but it would be nice if your firmware could calibrate the radio.  Instead of having to temporarily load this other firmware.

**Comment by ErikS-web** [comment]
Just read #52

So what do you think ?

**Reply by ErikS-web** [reply]
Why calibrate the output. 
And then it **_only applies to the one frequency to which you have calibrated it._** 
Provided you have a good power meter.

I have a lot of different equipment and is it nowhere the same on any of my equipment and it doesn't matter to me. 
It doesn't all happen exactly like that.

Only if I want to take measurements do I use an (adjustable) attenuator in the antenna line, to limit the power.

**Comment by ErikS-web** [comment]
Some realistic power measurements....
Not clear which firmware is used. Some on UVMOD based on 1.26

But he has the right measuring equipment to demonstrate.
The prove what the radio is designed for.

*******************************************
Watch from the beginning... no...
measurements start here...
https://www.youtube.com/watch?v=uqINxMXyetA

Interesting part comes from here

many harmonics
https://youtu.be/uqINxMXyetA?t=855

only at 144 it is OK
https://youtu.be/uqINxMXyetA?t=882

at 433
https://youtu.be/uqINxMXyetA?t=939
and average power is higher power ;-)

power measurements
https://youtu.be/uqINxMXyetA?t=1052

**Reply by jclagreca** [reply]
The reason to calibrate is if some previous firmware has messed up your values.  You will want to calibrate the radio back to what it was designed for.  

[https://bit.ly/417mVxA](https://bit.ly/417mVxA)

![MCFWV1 0CAL](https://github.com/egzumer/uv-k5-firmware-custom/assets/64165889/08172bb9-c857-4480-b4b5-5f45bace964a)
You can download MCFW_V1.0_CAL.bin firmware to calibrate here:  [https://bit.ly/3NbQE2u](https://bit.ly/3NbQE2u)

**Comment by ErikS-web** [comment]
Then the solution is VERY SIMPLE ... If that ever happens.

Then use that firmware to set things up properly again.
But read the instructions carafully.... _But where are the instructions ?_

And I assume this requires a adequate powermeter

**Reply by jclagreca** [reply]
Where are the instructions to any of this? How will you know if that ever happens and you need to apply the calibration?  Most of us don't have calibration equipment available to us.  Which is why I suggest running the calibration firmware just before loading your favorite firmware for long term use.

**Comment by ErikS-web** [comment]
DID you try and measure? Then please join us in your experience.

Simply loading other firmware without additional actions and such as calibration with an external and correct power meter is all an illusion of accuracy in my opinion .... "Showing something on display".

Please show the manual/instructions and preferably also the source code. So that it can be analyzed what the intention of implementation is.
